.github/workflows/job-postcommit-python-io-kafka.yml

```diff
@@ -0,0 +1,75 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Test for Python Postcommit IO Kafka
+
+
+name: Python Postcommit IO Kafka
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+    paths: [ 'sdks/python/apache_beam/io/external/xlang_kafkaio_it_test.py']
```

Comment:
Postcommits probably shouldn't be conditional on paths, they should run no matter what

Reply:
Since different tests are in the path `sdks/python/apache_beam/io/external/` (and there are other workflows associated as well to each test in this path), we are considering that every time that specific path/file is changed, it should trigger the workflow.